security/pkg/nodeagent/cache/helper.go

```diff
@@ -75,6 +75,12 @@ func isRetryableErr(c codes.Code, httpRespCode int, isGrpc bool) bool {
 
 // cacheLogPrefix returns a unified log prefix.
 func cacheLogPrefix(conID, resourceName string) string {
-	lPrefix := fmt.Sprintf("node:%s resource:%s", conID, resourceName)
+	lPrefix := fmt.Sprintf("[Node:%s Res:%s]", conID, resourceName)
```

Comment:
why are we inventing new logging formats?

Reply:
Good question. I think this makes the logging line shorter (more concise). But if this pattern aligns with other components. I can change it back.